### ISSUE TITLE
Question regarding the sources of the images

### ISSUE THREAD

**Issue by WentaoTan** [yuhui-zh15/AutoConverter#3]
I find VMCBench to be very insightful. As you mentioned, it is a benchmark created by "transforming 20 existing VQA datasets into a unified multiple-choice format, totaling 9,018 questions. "

I have a question regarding the sources of the images and questions in VMCBench. Are they taken from the test sets or the training sets of these 20 VQA datasets? My concern is that if they are sourced from the training sets, the models being evaluated may have already seen these samples. On the other hand, if they are sourced from the test sets, does this mean the distractor choices of the original test samples have simply been modified?

I would greatly appreciate your clarification. Thank you so much for your time and help!

**Comment by yuhui-zh15** [comment]
Hi Wentao, sorry I missed this question, and thanks for your interest in our work! Images are taken from the test set, and the distractor choices are re-generated using AutoConverter. Let me know if you have any other questions!
